### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by pornel**

### CODE REVIEW

**Comment by zorgiepoo** [comment]
If we recommend using signed dmg's, then shouldn't the "If you distribute your app as a disk image (DMG):" section be above the "if you are using a zip/tar" section?

If you use a zip/tar, you can't archive *more than one item*. DMG's allow multiple top level items, zip archives do not. That wording needs to be changed.

I think I'm actually okay with LetsMove being mentioned if the app developer distributes a zip/tar despite what I said in #1012.. I didn't read it carefully enough, my bad.

Are you planning to keep it to suggest using the same archive, or not using dmg for shipping Sparkle updates?

File: documentation/index.md [hunk 1]
```diff
@@ -11,19 +11,30 @@ Note that Sparkle does [not yet support](//github.com/{{ site.github_username }}
...
+  * Encourage users to move the app to `/Applications`, e.g. use [LetsMove](https://github.com/potionfactory/LetsMove/). The system will not allow the app to update itself until it's moved.
+  * Avoid placing your app inside another folder in your archive, because copying of the folder as a whole doesn't remove the quarantine.
 
-These guidelines follow Apple's best practices and works best with App Translocation which affects macOS 10.12 and later. When an application is running from a dmg or is translocated, Sparkle will not be able to update your application. Therefore, you should distribute your application in a way that will most likely be moved by the user.
+If you distribute your app as a disk image (DMG):
+
+  * Add an `/Applications` symlink in your DMG, or otherwise encourage the user to copy the app out of it (the app can't be updated when it's launched straight from the disk).
+  * Make sure the DMG signed with a Developer ID and use macOS 10.11.5 or later to sign it (an older OS may not sign correctly). Signed DMG archives are backwards compatible.
```
Comment (zorgiepoo) [inline documentation/index.md]: Make sure the DMG *is* signed

**Comment by zorgiepoo** [comment]
Thinking back on this, as far as "We recommend using Cocoapods"... I think it's important to consider:

* Cocoapods adds a not-small tooling cost to the Xcode project
* Users of Cocoapods may already be using it despite our recommendation (i.e, if they are already using it for other deps, they will be using it anyway)
* Cocoapods doesn't directly solve the issue of being notified when newer versions being available

I think it's fine to just say "If you use CocoaPods" and leave it mentioned first.

(When to upgrade Sparkle, and how often a developer may want to upgrade it, is a different issue. Careful not to fall under mindset that "always upgrade as fast as possible" is always the best route..)

Then there's still directly mentioning LetsMove which I guess I'm still hesitant about (there's a reason why I used the wording "or otherwise encourage the user to copy the app out of it"). Don't want to come across in a way of giving official approval of their quality of code, or reliability. Part of this issue is discussed in the other linked issue. It is also dwarfed a bit by the recommendation to use a signed dmg.

(Will cc @pornel since this issue was prematurely merged, and noticed you aren't watching this repo, and I'm not sure if the notifications persist)

**Comment by pornel** [comment]
I've made the text changes. 

Do you know any better solution than LetsMove? If we don't recommend it, then people will either do nothing (which breaks update and makes Sparkle show inaccurate error), use LetsMove, or invent their own thing which is unlikely to even handle cases that LetsMove handles.

**Comment by zorgiepoo** [comment]
I think these two issues from my feedback are still relevant:

> If we recommend using signed dmg's, then shouldn't the "If you distribute your app as a disk image (DMG):" section be above the "if you are using a zip/tar" section?

> If you use a zip/tar, you can't archive more than one item. DMG's allow multiple top level items, zip archives do not. That wording needs to be changed.

> Do you know any better solution than LetsMove? If we don't recommend it, then people will either do nothing (which breaks update and makes Sparkle show inaccurate error), use LetsMove, or invent their own thing which is unlikely to even handle cases that LetsMove handles.

Not really. I guess I'm more comfortable with doing nothing and breaking update for some users more than officially approving LetsMove specifically. It makes sense saying "you should really use a signed dmg as Apple recommends." Then if we follow with "but if you use zip/tar which is not recommended..." I guess it's questionable whether it's worth at that point to bring up LetsMove specifically.

**Comment by pornel** [comment]
> If you use a zip/tar, you can't archive more than one item. 

I thought Sparkle does support that. We use zip/tar command line, so it doesn't create an extra folder like the GUI apps do, and we scan the folder for an app with matching ID (`SUInstaller installSourcePathInUpdateFolder:…`)

**Comment by pornel** [comment]
I've moved the DMG recommendation first.

I've tried adding something like "don't try to copy your app to Applications yourself, but if you must, use LetsMove, as it handles more edge cases", but that only sounds like a stronger recommendation.

I would still rather mention LetsMove than leave people to reinvent it themselves. LetsMove handles reasonable  number of edge cases and the code doesn't look worse than Sparkle a couple of versions ago :)

**Comment by zorgiepoo** [comment]
> I thought Sparkle does support that. We use zip/tar command line, so it doesn't create an extra folder like the GUI apps do, and we scan the folder for an app with matching ID (SUInstaller installSourcePathInUpdateFolder:…)

This has to with app translocation and initial distribution, not Sparkle. If you archive multiple items using zip or tar you create an intermediate folder (extracted via Finder) that the user may move instead of moving the app in the folder.

**Pull request merged**